src/elements/button/button.js

```diff
@@ -2,7 +2,7 @@
 'use strict';
 
 angular
-  .module('semantic.ui.elements.button', [])
+  .module('semantic.ui.elements.button', ['semantic.ui.elements.icon', 'semantic.ui.elements.divider'])
```

Comment:
It's a safe bet to always create a new branch for any feature or fix you're working on, no matter how small